pkg/persistence/db.go

```diff
@@ -34,7 +34,7 @@ type Conn interface {
 	//
 	// Only returns keys that correspond to a Key for a date
 	// less than 14 days ago.
-	FetchKeysForDateNumber(string, uint32, int32) ([]*pb.TemporaryExposureKey, error)
+	FetchKeysForHours(string, uint32, uint32, int32) ([]*pb.TemporaryExposureKey, error)
```

Comment:
Any reason to remove FetchKeysForDateNumber()?
Would it be sensible to keep it around and have it be a wrapper around FetchKeysForHours()?

Actually, I just checked in retrieve() in retrieve.go and it doesn't really seem like it would actually simplify the logic. Since it seems to be the only place we're using this there probably isn't a benefit.

Reply:
I think it adds a level of abstraction that we may not need, we can always bring these back